storyStarterGame.js

```diff
@@ -0,0 +1,321 @@
+        darkWoodsStoryLine();
+    } else {
+        leaveGame();
+    }
+
+}
+
+// romanceStory
+const romanceStory = () => {
+     let characterNumRom = readLineSync.question ("How many character you want in this story?") 
+    if (characterNumRom > 3) {
+        tooManyCharRom();
+    } else {
+        perfectNumRom();
+    }
+}
+
+//romSetting
+const romSetting = () => {
+    let romanceSetting = readLineSync.question ("Do you want to start your story in the city or in a home?")
+    if (romanceSetting === "city") {
+        cityDate()
+    } else if (romanceSetting === "home") {
+        stayAtHomeDate()
+    } else {
+        leaveGame()
+    }
+}
+    
+//perfectNum
+const perfectNumRom = () => {
+    console.log ("Perfect!!!")
+    console.log ("You will now have to choose your storyline!")
+    let romSetting = readLineSync.question ("Do you want to start your story in the city or at home?")
+    if (romSetting === "city") {
+        console.clear();
+        cityDate();
+    } else if (romSetting === "home") {
+        stayAtHomeDate();
+    } else {
+        leaveGame();
+    }
+}
+//tooManyCharRom
+const tooManyCharRom = () => {
+    let characterNum = readLineSync.question ("How many character you want in this story?") 
+    if (characterNum <= 2) { 
+        console.clear();
+        perfectNumRom();
+    } else {
+        console.log ("Seriously?! Please choose less than 3! This is not a tella novela! You want to build on them.")
+        tooManyCharRom();
+    } 
+}
+
+//cityDate
+const cityDate = () => {
+   let charMainName = readLineSync.question("What's your main character's name?")
+     console.log(`${charMainName} is on a date! The first time in so long! \nWalking through the city and enjoying their date's company. \nNow they're getting hungry from all the walking and talking! \n"Let's grab something to eat", says ${charMainName}`)
+        array = ["5 star resturant", "street vendors", "home-cooked meal"]
+       let eatingDate = readLineSync.keyInSelect (array)
+       let youChooseFood = array[eatingDate] 
+        if (youChooseFood === "5 star resturant") {
+        fiveStarResturant ();
+    } else if (youChooseFood === "street vendors") {
+        streetVendor();
+    } else if (youChooseFood === "home-cooked meal") {
+        homeCooked();
+    } else if (youChooseFood === undefined){ 
+        if (readLineSync.keyInYN("Would you like to choose another story?")) {
+            startGame();
+        } else {
+            leaveGame();
+        }
+
+    }
+    
+}
+// fiveStarResturant
+const fiveStarResturant = () => {
+        console.log("Your character ate bad oyster! Your character got food poisioning!")
+        for (i = 0; i < array.length; i++) {
+            if (array[i] === "5 star resturant") {
+                array.splice(i,1)
+            }
+        }
+        let eatingDate = readLineSync.keyInSelect (array)
+       let youChooseFood = array[eatingDate]
+       if (youChooseFood === "street vendor"){
```

Comment:
When I played this part I was confused because you didn't say that I was picking another date. It was just I got food poisoning and was then eating again

Reply:
Sorry you got food poisoning. My intent was to take that function into another one. However I kept all of my focus elsewhere. I will fix this!